### DISCUSSION TITLE
Billing API

### DISCUSSION THREAD

**Discussion by matmicro** [codemagic-ci-cd/codemagic-docs#2886] [Ideas]
### Use case

```Text
As a codemagic user, i would like to be able to access my invoices from REST API.
Thus i could automate retrieving of:
- bill amount
- bill tax
- bill reference
- status (pending/paid)
- link to download the PDF file
```

### Idea

```Text
Add a new routes to actual REST API.
https://docs.codemagic.io/rest-api/codemagic-rest-api/
```

**Comment by mikhail-tokarev** [comment]
Hi @matmicro thank you for the idea. I'll keep it open to see if there are more people who needs this functionality

**Comment by icarusdust** [comment]
Hi @matmicro, if it suits you, I'd like to discuss your requirements closely. Please book a time [here](https://meetings-eu1.hubspot.com/nihal-aghazade/codemagic-strategy).  This will help us evaluate the request faster.
